app/mailers/user_mailer.rb

```diff
@@ -36,4 +36,13 @@ def coordinator_matched_recipient(donation, donor_id, recipient_id)
     @recipient = Recipient.find recipient_id
     mail(to: @recipient_profile.contact_email, subject: "(Recipient) Match made!")
   end
+  
+  def match_canceled(donation, recipient_id, coordinator_id)
```

Comment:
Unused method argument - `coordinator_id`. If it's necessary, use `_` or `_coordinator_id` as an argument name to indicate that it won't be used.